src/shared/clients/codecatalystClient.ts

```diff
@@ -445,12 +467,17 @@ class CodeCatalystClientInternal {
 
     /**
      * Lists ALL of the given resource in the current account
+     *
+     * @param firstPartOnly If true, only first party repos (code catalyst owned) will be returned.
```

Comment:
CodeCatalyst is one word

```suggestion
     * @param firstPartOnly If true, only first party repos (CodeCatalyst owned) will be returned.
```